eland/field_mappings.py

```diff
@@ -537,7 +535,17 @@ def _generate_es_mappings(
         }
         """
 
-        mapping_props = {}
+        mapping_props: Dict[str, Any] = {}
+        es_dtype: str
```

Comment:
Is this type hint necessary? If it is let's move it to the top of the function with whitespace before the `mapping_props = {}` so we don't confuse it for an initialization.

Reply:
Just added, Anyway will move it as suggested